modes.xml

```diff
@@ -119,4 +119,3 @@
     </pipeline>
   </mode>
 </modes>
-
```

Comment:
It's conventional to leave an empty line at the end of source files, e.g. so that if I do `cat modes.xml`, my terminal prompt doesn't end up at the end of `modes.xml`, but on a new line. Same comment applies to the previous file.

Reply:
oh thanks for this information @sushain97 , now should I make a new PR after changing this?

Reply:
nah, push to the same branch and those commits are auto-added to this PR